app/routes/technical-help.js

```diff
@@ -12,65 +12,66 @@ module.exports = function (router) {
-          req.body['date-of-claim-month'],
-          req.body['date-of-claim-year'],
-          req.body.issue
-        )
+      const technicalHelp = new TechnicalHelp(
+        req.body.name,
+        req.body.emailAddress,
+        req.body.referenceNumber,
+        req.body['date-of-claim-day'],
+        req.body['date-of-claim-month'],
+        req.body['date-of-claim-year'],
+        req.body.issue
+      )
+
+      const formattedDate = technicalHelp.dateOfClaim ? `${req.body['date-of-claim-day']}-${req.body['date-of-claim-month']}-${req.body['date-of-claim-year']}` : ''
 
-        const formattedDate = technicalHelp.dateOfClaim ? `${req.body['date-of-claim-day']}-${req.body['date-of-claim-month']}-${req.body['date-of-claim-year']}` : ''
+      let subjectText = 'Help With Prison Visits - Help'
+      const tagText = ['HelpWithPrisonVisits']
 
-        let subjectText = 'Help With Prison Visits - Help'
-        const tagText = ['HelpWithPrisonVisits']
+      if (config.ZENDESK_TEST_ENVIRONMENT === 'true') {
+        subjectText = 'Test: Help With Prison Visits - Help'
+        tagText.push('Test')
+      }
 
-        if (config.ZENDESK_TEST_ENVIRONMENT === 'true') {
-          subjectText = 'Test: Help With Prison Visits - Help'
-          tagText.push('Test')
+      const ticket = {
+        ticket: {
+          submitter_id: '114198238551',
+          requester: {
+            name: technicalHelp.name,
+            email: technicalHelp.emailAddress,
+            verified: true
+          },
+          subject: subjectText,
+          comment: {
+            body: `Reference number: ${technicalHelp.referenceNumber}\n\nDate of Claim: ${formattedDate}\n\n${technicalHelp.issue}`
+          },
+          tags: tagText
         }
+      }
 
+      if (config.ZENDESK_ENABLED === 'true') {
         const zendeskApiUrl = `${config.ZENDESK_API_URL}/api/v2/tickets.json`
         const credentials = `${config.ZENDESK_EMAIL_ADDRESS}/token:${config.ZENDESK_API_KEY}`
         const headers = {
           Authorization: 'Basic ' + Buffer.from(credentials).toString('base64')
         }
 
-        const ticket = {
-          ticket: {
-            submitter_id: '114198238551',
-            requester: {
-              name: technicalHelp.name,
-              email: technicalHelp.emailAddress,
-              verified: true
-            },
-            subject: subjectText,
-            comment: {
-              body: `Reference number: ${technicalHelp.referenceNumber}\n\nDate of Claim: ${formattedDate}\n\n${technicalHelp.issue}`
-            },
-            tags: tagText
-          }
-        }
-
         return axios.post(zendeskApiUrl, ticket, { headers })
           .then(function (response) {
             if (response.status === 201) {
               log.info(`Zendesk ticket ${response.data.ticket.id} has been raised`)
             }
 
-            return res.redirect('/')
+            return res.redirect('/start')
```

Comment:
Changed this because the form wasn't submitting properly on Chrome because of new default CSP since helmet was recently upgraded to v5 (same change that broke it working locally on Safari). Problem is `form-action` policy - details [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/form-action), it's a bit inconsistent between browsers. Problem is that redirecting to `'/'` then redirects to a different domain which is then blocked. Took a while to figure this out! Obviously a better longer term fix would be to have something like a confirmation page acknowledging form submission, but felt that was a bit outside the scope of this work...